LICENSE.md

```diff
@@ -0,0 +1,105 @@
+# Functional Source License, Version 1.1, Apache 2.0 Future License
+
+## Abbreviation
+
+FSL-1.1-Apache-2.0
+
+## Notice
+
+Copyright 2008-2024 Functional Software, Inc. dba Sentry
```

Comment:
```suggestion
Copyright 2024 Cirrus Labs, Inc.
```

Reply:
Please take a look at https://github.com/getsentry/fsl.software/blob/main/FSL-1.1-Apache-2.0.template.md:

>Copyright ${year} ${licensor name}

Why would they make this line configurable then?